Management_System/DAL/Entities/BoardEntity.cs

```diff
@@ -7,6 +7,6 @@ public class BoardEntity
     public string Name { get; set; } = string.Empty;
     public string Description { get; set; } = string.Empty;
 
-    public IEnumerable<TableEntity> Tables { get; set; } = [];
-    public IEnumerable<UserBoardEntity> UserBoards { get; set; } = [];
+    public IEnumerable<TableEntity> Tables { get; set; } = new List<TableEntity>();
+    public IEnumerable<UserBoardEntity> UserBoards { get; set; } = new List<UserBoardEntity>();
```

Comment:
Why did you do that?

Reply:
I had "System.Text.Json.JsonException" when I tried to get the board by id

Reply:
How it is related to the selected lines?